cookbooks/travis_build_environment/recipes/mysql.rb

```diff
@@ -60,6 +75,16 @@
     > SET PASSWORD FOR 'root'@'localhost' = PASSWORD('');
     > SET PASSWORD FOR 'root'@'127.0.0.1' = PASSWORD('');
   EOF
+  not_if { mysql_version == 5.7 }
+end
+
+file mysql_users_passwords_sql do
+  content <<-EOF.gsub(/^\s+> /, '')
+    > SET old_passwords = 0;
+    > CREATE USER 'travis'@'%' IDENTIFIED BY '';
+    > SET PASSWORD FOR 'root'@'localhost' = PASSWORD('');
+  EOF
+  only_if { mysql_version == 5.7 }
```

Comment:
Ah, OK.  We've very recently added more statements to the other `file` resource.  Can you update this `file` resource so that it only excludes the statement that you need skipped on 5.7?